packages/esm-patient-conditions-app/src/index.ts

```diff
@@ -29,7 +29,7 @@ function setupOpenMRS() {
       {
         name: 'conditions-overview-widget',
         slot: 'patient-chart-summary-dashboard-slot',
-        order: 6,
+        order: 3,
```

Comment:
It would be nice if we had a distinct order per widget. Both this and medications are at order 3.

Reply:
Thanks for the catch!